setup.cfg

```
@@ -32,6 +33,8 @@ install_requires =
 [options.extras_require]
 tests =
     invenio-app>=1.3.4,<2.0.0
+    invenio-celery>=1.2.5
+    invenio_mail>=1.0.2
```

```suggestion
    invenio-mail>=1.0.2
```

minor: unless we've made a typo on the PyPI release :)